packages/uniforms/src/useField.tsx

```diff
@@ -18,7 +18,10 @@ function propagate(
   const resultValue =
     typeof prop === 'string'
       ? prop
-      : prop === false || (prop === undefined && !state) || schemaDisabled
+      : prop === null ||
+        prop === false ||
+        (prop === undefined && !state) ||
+        schemaDisabled
```

Comment:
Can we extract it to function with descriptive name?

Reply:
Will do when we'll come up with one.